resources/knative-build/values.yaml

```
@@ -0,0 +1,3 @@
+global:
+  isLocalEnv: false
+  knative: false
```

It should be enabled.

> 
```suggestion
  knative: true
```